segmentation/utils/spatial_analysis_utils_test.py

```diff
@@ -0,0 +1,314 @@
+        dist_mat_pos[:10, 10:20] = 50
+        dist_mat_pos[20:40, :20] = 200
+        dist_mat_pos[:20, 20:40] = 200
+        dist_mat_pos[40:80, :40] = 300
+        dist_mat_pos[:40, 40:80] = 300
+        return dist_mat_pos
+    elif typeofenfrichment == "negative":
+        # This creates a distance matrix where there are two groups of cells significant for 2 different
+        # markers that are not located near each other (not within the dist_lim).
+        dist_mat_neg = np.zeros((60, 60))
+        dist_mat_neg[20:40, :20] = 300
+        dist_mat_neg[:20, 20:40] = 300
+        dist_mat_neg[40:50, :40] = 50
+        dist_mat_neg[:40, 40:50] = 50
+        dist_mat_neg[50:60, :50] = 200
+        dist_mat_neg[:50, 50:60] = 200
+        return dist_mat_neg
+
+
+def make_expression_matrix(typeofencrichment):
+    # Create the expression matrix with cell labels and patient labels for no enrichment,
+    # positive enrichment, and negative enrichment.
+
+    if typeofencrichment == "none":
+        all_patient_data = pd.DataFrame(np.zeros((60, 31)))
+        # Assigning values to the patient label and cell label columns
+        all_patient_data[30] = "Point8"
+        all_patient_data[24] = np.arange(len(all_patient_data[1])) + 1
+        # We create two populations of 20 cells each, each positive for a different marker (column index 2 and 3)
+        all_patient_data.iloc[0:20, 2] = 1
+        all_patient_data.iloc[20:40, 3] = 1
+        # Assign column names to columns not for markers (columns to be excluded)
+        all_patient_data = all_patient_data.rename({0: 'cell_size', 1: 'Background', 14: "HH3",
+                                                    23: "summed_channel", 24: "label", 25: "area", 26: "eccentricity",
+                                                    27: "major_axis_length", 28: "minor_axis_length", 29: "perimeter",
+                                                    30: "fov"}, axis=1)
+        return all_patient_data
+    elif typeofencrichment == "positive":
+        all_patient_data_pos = pd.DataFrame(np.zeros((80, 31)))
+        # Assigning values to the patient label and cell label columns
+        all_patient_data_pos[30] = "Point8"
+        all_patient_data_pos[24] = np.arange(len(all_patient_data_pos[1])) + 1
+        # We create 8 cells positive for column index 2, and 8 cells positive for column index 3.
+        # These are within the dist_lim in dist_mat_pos (positive enrichment distance matrix).
+        all_patient_data_pos.iloc[0:8, 2] = 1
+        all_patient_data_pos.iloc[10:18, 3] = 1
+        # We create 4 cells in column index 2 and column index 3 that are also positive
+        # for their respective markers.
+        all_patient_data_pos.iloc[28:32, 2] = 1
+        all_patient_data_pos.iloc[32:36, 3] = 1
+        # Assign column names to columns not for markers (columns to be excluded)
+        all_patient_data_pos = all_patient_data_pos.rename({0: 'cell_size', 1: 'Background', 14: "HH3",
```

Comment:
You can make a dict called col_names, and define it as equal to everything in brackets. Then, for each of these pandas arrays, call .rename(col_names, axis=1). That way you don't have to retype it